src/librbd/api/Group.cc

```diff
@@ -54,35 +55,15 @@ snap_t get_group_snap_id(I* ictx,
 }
 
 int group_snap_list(librados::IoCtx& group_ioctx, const std::string& group_id,
-		    std::vector<cls::rbd::GroupSnapshot> *cls_snaps)
+                    std::vector<cls::rbd::GroupSnapshot> *cls_snaps,
+                    bool sort = false, bool require_sort = false)
```

Comment:
I'd suggest switching to more descriptive names that were used in https://github.com/ceph/ceph/pull/58002:
- `sort` -> `try_to_sort`
- `require_sort` -> `fail_if_not_ordered`

and also getting rid of defaults here -- being explicit at the call site is better, especially when it comes to more than one bool.

Reply:
Changed to try_to_sort and fail_if_not_sorted.